docker/dockerfiles/Dockerfile.nethermind

```diff
@@ -0,0 +1,11 @@
+FROM nethermind/nethermind:1.31.6
```

Comment:
Is there a specific reason for not using the `latest` tag but a hardcoded version? Do we need to update this with each release?

Reply:
Execution client may not working if it's upgraded ahead of op-node.